cve_test.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func TestFetchCVEv2(t *testing.T) {
-	cliv2 := NewClientV2()
+	cliv2 := NewClientV2(os.Getenv("NVD_API_KEY"))
```

Comment:
this is where I changed the test so we would read the API key from an env variable.